apps/connect/package.json

```diff
@@ -30,7 +30,9 @@
     "@mui/icons-material": "^5.14.11",
     "@mui/material": "^5.12.1",
     "@tanstack/react-query": "^5.14.2",
-    "@wormhole-foundation/wormhole-connect": "^0.3.17",
+    "@wormhole-foundation/wormhole-connect": "file:wormhole-foundation-wormhole-connect-0.3.7.tgz",
```

Comment:
this should not be in the prod release, right?

Reply:
No, this is only to test in preview